### PR TITLE
Various tweaks and Medical Bay additions, alieviates my anxiety

### PR DESCRIPTION
**Pull request by sheeplets**
A laundry list of minor changes which I lose sleep over.

- Organizes things on a few tables to look cleaner. The store room is now organized, no more turf-tab hunting.
- Removes an out of place ICU floor decal in the GTR.
- Removes the duplicate glove boxes from the OR's, adds another box with the others across from the elevator. Along with surgical masks and a second box of body bags. I like even numbers.
- Removes the racks with safety goggles on them from inside the OR's.
- Relocates one bag of O- blood from each OR to the ICU's supply. Bringing it to a total of six bags (up from four).
- Changes the disposals chute in the Psychiatrist's office from a full-sized one to the slim variant, to prevent further suicide attempts. Per request.
- Removes water cups from Medical and the Brig. You can click the coolers with an empty hand to dispense an infinite number of cups.
- Moves the ringer button in Medical's lobby to just outside the GTR's front door.
- Adds a first aid wall closet into the Medbay lobby. Two stacks of gauze, two stacks of ointment, two inaprovaline injectors, and two dylovene injectors. Your odds of survival during deadpop have very slightly improved.
- Adds a bottle of bicaridine tablets to the medication cabinet in the GTR and First Responders' quarters. For chemically treating AB without surgeons.
- Adds another coagzolug autoinjector to the First Responders' quarters, a total of four (up from three).
- Allows the Rescue Hardsuit to hold blood bags in the suit storage slot. Per request.
- Adds two lifeform stasis beds to the ICU, nerfed their stasis factor (15x → 10x) to be in line with sleepers. They were a little _too_ good.
- Added a cheeky little window behind a status board to get rid of the blind spot into the ICU.

### CODE REVIEW

**Comment by sheeplets** [comment]
!review

**Comment by EmperorJayJay** [comment]
I am an enjoyer of this.

**Comment by sheeplets** [comment]
Wasn't aware of the hold on mapping PRs, closing this for the time being.

File: html/changelogs/sheeplets.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - tweak: "Made various mapping tweaks to the Medical Bay."
```
Comment (SleepyGemmy) [inline html/changelogs/sheeplets.yml:41]:
```suggestion
  - maptweak: "Made various mapping tweaks to the Medical Bay."
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request closed without merging**